packages/paste-core/components/toast/stories/index.stories.tsx

```diff
@@ -231,6 +231,9 @@ export const ToasterStory = (): React.ReactNode => {
       variant,
       message: messageText,
       ...(toastTimeout !== '0' && {dismissAfter: Number.parseInt(toastTimeout, 10)}),
+      onDismiss: () => {
+        console.log('dismissed!');
+      },
```

Comment:
Would it be helpful to have the story do something visible on the UI rather than console logging, so that it'll be easy to see this happening if we use it as an example on the website?

Reply:
I don't think it's necessary for this particular prop. It's a pretty niche requirement of Toasts.